AllReadyApp/Web-App/AllReady/ViewModels/ActivityViewModel.cs

```diff
@@ -45,6 +48,11 @@ public ActivityViewModel(Activity activity)
                  : new List<TaskViewModel>();
 
             RequiredSkills = activity.RequiredSkills?.Select(acsk => acsk.Skill).ToList();
+
+            if(!string.IsNullOrEmpty(activity.Campaign.ManagingOrganization.PrivacyPolicy))
+            {
+                HasPrivacyPolicy = true;
+            }

```

Comment:
Simplify this as `HasPrivacyPolicy = !string.IsNullOrEmpty(activity.Campaign.ManagingOrganization.PrivacyPolicy)` here and in the other view models.